test/features/kogito_service_performance.feature

```diff
@@ -73,8 +77,8 @@ Feature: Kogito Service Performance
       }
       """
 
-    Then Service "process-quarkus-example" contains <requests> instances of process with name "orders" within 1 minutes
-    And Service "process-quarkus-example" contains <requests> instances of process with name "orderItems" within 1 minutes
+    #Then Service "process-quarkus-example" contains <requests> instances of process with name "orders" within 1 minutes
```

Comment:
why is it commented out?

Reply:
Please see the point no. 4 in the PR description.